nio_cli/commands/config.py

```diff
@@ -26,8 +28,61 @@ def config_project(name='.'):
                 tmp.write('PK_TOKEN={}\n'.format(pk_token))
             else:
                 tmp.write(line)
-        os.remove(env_location)
-        os.rename(tmp.name, env_location)
+        os.remove(conf_location)
```

Comment:
I think these need to exist outside of the context manager (`with`) based on https://github.com/niolabs/nio-cli/pull/76